### PR TITLE
Fix trailing duplicate prefixes issue

### PR DESCRIPTION
**Pull request by cnlinh**
## What this does
This PR
1. fixes the issues where the system would parse the second prefix instead of the first one (e.g. `alias a/h cmd/help cmd/clear` would map `h` to `clear` instead of `help cmd/clear` as an expected behavior.

## How to test
1. Launch SunRez.
2. Execute the `alias a/h cmd/help cmd/clear` command with the name of the previously created alias. 
3. Execute the alias. You should see a command not found error.
4. Check the preference json file to see that `h` is mapped to `help cmd/clear`.

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/alias/AliasCommandParser.java [hunk 1]
```diff
@@ -19,14 +23,15 @@ public AliasCommand parse(String args) throws ParseException {
...
-        String aliasName = argMultimap.getValue(PREFIX_ALIAS).orElse("");
-        String command = argMultimap.getValue(PREFIX_COMMAND).orElse("");
-
-        if (aliasName.isEmpty() || command.isEmpty()) {
+        if (argMultimap.getValue(PREFIX_ALIAS).isEmpty() || argMultimap.getValue(PREFIX_COMMAND).isEmpty()) {
             throw new ParseException(
                     String.format(MESSAGE_INVALID_COMMAND_FORMAT, AliasCommand.MESSAGE_USAGE));
         }
 
+        String[] parsedArgs = args.split("\\s+", 3);
```
Comment (stein414) [inline src/main/java/seedu/address/logic/parser/alias/AliasCommandParser.java:31]: Perhaps abstract the 3 into a variable to avoid magic numbers

**Review by stein414 — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/logic/parser/alias/AliasCommandParser.java (hunk 1 above)
Comment (cnlinh) [reply src/main/java/seedu/address/logic/parser/alias/AliasCommandParser.java:31]: Good point!

**Review by DrWala — APPROVED** [review]
The fix works as expected.

Would suggest updating the UG to reflect how this behaviour works—Stating that anything after the cmd will be interpreted as a command outright.

**Review by colintkn — APPROVED** [review]

**Review by stein414 — APPROVED** [review]
LGTM

**Pull request merged**